PSReadLine/Suggestion.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+
+namespace Microsoft.PowerShell
+{
+    public partial class PSConsoleReadLine
+    {
+        private string _suggestionText;
+        private string _userInputText;
```

Comment:
This field needs a better name or a good comment. I could get confused and think it's the current command line.

Reply:
Will rename the field.